magefile.go

```diff
@@ -98,8 +98,8 @@ func porter(args ...string) sh.PreparedCommand {
 
 // Run end-to-end (e2e) tests
 func TestE2E() error {
-	mg.Deps(startLocalDockerRegistry)
-	defer stopLocalDockerRegistry()
+	mg.Deps(StarlDockerRegistry)
```

Comment:
curious about the StarlDockerRegistry name from startLocalDockerRegistry .. Should it be StarLDockerRegistry? Just confused here about the naming. I see that the function is defined below and it matches but I wasn't sure if the L stood for local or if this was something else so the camel casing seems weird to me.

Reply:
Maybe it was meant to be StartDockerRegistry to match up with the StopDockerRegistry..

Reply:
Yeah it was a typo. I'll submit a PR to fix that! 🤦‍♀️

Reply:
https://github.com/getporter/porter/pull/1423